pil-analyzer/src/condenser.rs

```diff
@@ -129,15 +138,15 @@ impl<T: FieldElement> Condenser<T> {
                 .into_iter()
                 .map(|constraint| {
                     Identity::from_polynomial_identity(
-                        identity.id,
```

Comment:
Is this really needed? I will re-write this part in #1187 anyway, so if it's working I would prefer no merge conflicts.

Reply:
I mean for #1154 I only care about identity IDs of lookups and permutations. So I guess it is fine as long as we don't return multiple lookups from functions. Should I undo it?

Reply:
Ah right! Ok, thanks for fixing!